.releaserc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   branches: [
     { name: "next" },
     { name: "next-rc", prerelease: "rc" },
+    { name: "pipeline-upload-assets", prerelease: "ignore-me2" },
```

Comment:
```suggestion
```

Reply:
only for testing the .tar.gz change, will be removed shortly